pkg/broker/types.go

```diff
@@ -10,6 +10,11 @@ type CatalogResponse struct {
 // ProvisionResponse is sent as the response to a provision call.
 type ProvisionResponse struct {
 	osb.ProvisionResponse
+
+	// AlreadyCreated - is used to if the request was already completed
```

Comment:
Did you mean set instead?

> is set if the request was already completed

Reply:
I honestly don't remember what I was attempting to say. your wording is 100% better though.